### PR TITLE
Add detail RPC requirements

### LINKED ISSUE #28: Add detail RPC requirement
- query account / address balance
- query sent / received transactions for an account / address
- query transaction details
- query fee / nonce for sending transaction
- query blockchain status (block height)
- send raw transaction

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #28

### CODE REVIEW

File: wallet-core/rpc-requirements.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] query transaction details
+- [ ] query fee / nonce for sending transaction
+- [ ] query blockchain status (block height etc)
+- [ ] send raw transaction
+
+## for blockatlas: 
+
+We need API or RPC to: 
+
+- [ ] query sent / received transactions for an account / address
```
Comment (vikmeup) [inline wallet-core/rpc-requirements.md]: link to blockatlas: `https://developer.trustwallet.com/blockatlas/newblockchain`

Comment (vikmeup) [inline wallet-core/rpc-requirements.md]:
```suggestion
- [ ] query sent / received list of transactions for an account / address
```

File: wallet-core/rpc-requirements.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] query transactions in a block
+
+## Example
+
+We will take XRP as an example here, it supports WebSocket, JSON-RPC (including over HTTP)
```
Comment (vikmeup) [inline wallet-core/rpc-requirements.md]:
```suggestion
We will take TRON as an example here, it supports WebSocket, JSON-RPC (including over HTTP)
```

File: wallet-core/rpc-requirements.md (hunk 1 above)
Comment (hewigovens) [reply wallet-core/rpc-requirements.md]: Added to above

**Pull request merged**